pkg/apis/apps/types.go

```diff
@@ -109,6 +153,17 @@ type StatefulSetSpec struct {
 	// all pods at once.
 	// +optional
 	PodManagementPolicy PodManagementPolicyType
+
+	// updateStrategy indicates the StatefulSetUpdateStrategy that will be
+	// employed to update Pods in the StatefulSet when a revision is made to
+	// Template or VolumeClaimsTemplate.
```

Comment:
Can we remove the VolumeClaimsTemplate from here, since we dont support the update to it.